Client/mods/deathmatch/logic/CClientGame.cpp

```diff
@@ -5582,6 +5602,39 @@ void CClientGame::ResetMapInfo()
     ReinitMarkers();
 }
 
+void CClientGame::SendObjectBreakPacket(CClientObject* Object, ElementID attackerID)
+{
+    if (Object && Object->GetHealth() == 0.0f)
+    {
+        NetBitStreamInterface* pBitStream = g_pNet->AllocateNetBitStream();
+        if (pBitStream)
+        {
+            pBitStream->Write(Object->GetID());
+            pBitStream->Write(attackerID);
+
+            g_pNet->SendPacket(PACKET_ID_OBJECT_BREAK, pBitStream, PACKET_PRIORITY_MEDIUM, PACKET_RELIABILITY_RELIABLE_ORDERED);
+            g_pNet->DeallocateNetBitStream(pBitStream);
+        }
+    }
+}
+
+void CClientGame::SendObjectDamagePacket(CClientObject* Object, float fLoss, ElementID attackerID)
```

Comment:
This is a mix of different notations. Please, select one of code styles and stick to it across the whole PR.